### PR TITLE
adding dev-env

### PR DESCRIPTION
**Pull request by akeiper**

### CODE REVIEW

**Review by jonyo — APPROVED** [review]

File: dev-tools.sh [hunk 1]
```diff
@@ -223,6 +223,29 @@ dev-xdebug-init() {
...
+
+	if [[ $config != '' ]]; then
+		local cmd="grep -i $config"
+	else
+		local cmd="cat"
+	fi
+
+	if [[ $service == service ]]; then
+		local path=/var/www/service/shared/config/environment.php
+	elif [[ $service == public ]]; then
```
Comment (ZurabWeb) [inline dev-tools.sh:240]: @akeiper What about `admin` and `api` ?

Comment (akeiper) [reply dev-tools.sh:240]: @ZurabWeb admin does not have shared

Comment (ZurabWeb) [reply dev-tools.sh:240]: @akeiper 

image

Comment (akeiper) [reply dev-tools.sh:240]: that is your local configured, this reads the shared

**Pull request merged**